buildspecs/buildspec-build-invoke.yml

```diff
@@ -0,0 +1,16 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      docker: 19
```

Comment:
Interesting, I don't even see a docker in runtime-versions [documentation](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#runtime-versions-buildspec-file) on code build. Does this even work?

Also why use docker only in this buildspec?

Reply:
+ Do we need specific docker version? Or this is required if you are using docker in your build?

Reply:
Good catch! I must found this somewhere, and this is deprecated! https://github.com/aws/aws-codebuild-docker-images/issues/283